web/src/components/PoisDesktop.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { PoiFeature } from 'api-client'
+
+import dimensions from '../constants/dimensions'
+
+const ListViewWrapper = styled.div<{ panelHeights: number }>`
+  min-width: 370px;
+  padding: 0 32px;
+  overflow: auto;
+  ${({ panelHeights }) => `height: calc(100vh - ${panelHeights}px - ${dimensions.toolbarHeight}px);`};
+`
+
+const ToolbarContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  background-color: ${props => props.theme.colors.backgroundAccentColor};
+  box-shadow: 1px 0 4px 0 rgba(0, 0, 0, 0.2);
+`
+
+const ListHeader = styled.div`
+  padding-top: 32px;
+  padding-bottom: 20px;
+  text-align: center;
+  font-size: 18px;
```

Comment:
somehow frustrating there is no appropriate fontSize in the theme for this.

Reply:
yea we have to refactor some typo stuff definitly but even think if we want to use css clamp for better responsiveness
https://developer.mozilla.org/en-US/docs/Web/CSS/clamp()